.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "singleQuote": true,
+  "jsxSingleQuote": true,
+  "semi": false,
```

Comment:
NIT: I prefer semi-colons, and the package currently uses semi's can we add them back in?

Reply:
Updated the config